src/linkActions.ts

```diff
@@ -66,6 +66,12 @@ export const linkActions = {
   talkToFlower: () => {
     displayMessageFromList('flower')
   },
+  dromadTam: () => {
```

Comment:
nit: Ideally we should remove older links too, because we're just kinda accumulating them.